packages/example-app-base/package.json

```diff
@@ -20,6 +20,7 @@
     "@types/chai": "3.4.35",
     "@types/enzyme": "2.8.0",
     "@types/es6-promise": "^0.0.32",
+    "@types/highlight.js": "^9.1.9",
```

Comment:
Can you remove the carets from these? What we've seen is that increasing the version can break the build unexpectedly, especially in cases like the react typings, which depend on the latest typescript. So I've removed the carets to ensure that we don't automatically pick up breaking changes for typings.